src/main/java/forestry/arboriculture/worldgen/TreeDecorator.java

```diff
@@ -87,6 +88,8 @@ public static void decorateTrees(World world, Random rand, int worldX, int world
 	@Nullable
 	private static BlockPos getValidPos(World world, int x, int z, ITree tree){
 		// get to the ground
+		if(Config.blacklistedTreeDims.contains(world.provider.getDimension()))
```

Comment:
Please move this check up to  decorateTrees(World, Random, int, int). It makes not much sense to check it every time if the world is every time the same.